packages/common/styles/_styles.scss

```diff
@@ -0,0 +1,13 @@
+@use './tokens';
+
+.dsco-text {
+  color: tokens.$dsco-font-color;
+  font-size: tokens.$dsco-font-size;
+  line-height: 20px;
+}
+
+.dsco-link {
```

Comment:
these styles could obviously be implemented in individual components, but i put them in this common styles file so that _any_ consumer can use these classes, even if they don't use our React components.
**upsides:**
- any consumer should be able to use these classes and their link will be styled appropriately
- extending styles across components is easier -- `<Link>` doesn't have to import/use `<Text>` in order to extend its styles

**downsides:**
- making a change in this package will re-publish `@dsco_/common` _and any consumers_. lerna does this automatically -- any package in this repo that depends on `@dsco_/common` will automatically get a patch-level version bump when `@dsco_/common` is published
- styles aren't co-located with corresponding components

Reply:
If we're trying to set up a pattern of being able to match the styles with the components, there will likely be some components this won't work for. I can imagine a component that has a bunch of complicated internal dom structure it requires for proper styling that wouldn't match a div with a class.

Reply:
I guess, we should be clear on how to use these classes and that they won't always result in a match with the actual component

Reply:
yeah, that's a great point. we would either need to commit to these classes being almost like a style API or be very clear about the fact that this isn't a 1:1 relationship. 

i think this file would be more useful as a set of utility classes, but it isn't obvious to me what those should be right now... maybe classes for text, headings, etc make more sense than a class for the link, which feels too specific right now and will likely have edge cases like you mentioned. if an organic need arises there, we can always move whatever shared styles we want into this package